lib/uwb/UwbPeer.cxx

```diff
@@ -32,6 +33,47 @@ UwbPeer::UwbPeer(UwbMacAddress address) :
     m_address(std::move(address))
 {}
 
+/**
+ * @brief Assuming the Arm definition of Qm.n formatting, the most significant bit is the sign, the next
+ * (m-1) bits are an integer, and the next n bits is the number to be multiplied by pow(2,n)
+ * The double equivalent will be the sum of those two results
+ * TODO double check this conversion, write tests for it
+ *
+ * @param q97 a number in Q9.7 format
+ * @return double
+ */
+double
+ConvertQ9_7FormatToIEEE(uint16_t q97)
```

Comment:
Please convert to camelCase.

Reply:
Should it be PascalCase instead? Also, I'm unsure how to format Q9.7 in PascalCase. I guess I could just do Q97 and hope users know that it doesn't mean a number with 2**(-97) precision.

Reply:
Yes sorry, PascalCase for function names. All that really matters is it starts with a capital and doesn't use "snakes", so `ConvertQ97FormatToIEEE` is good.